src/stories/beginners/add-remove-points/style.css

```
@@ -0,0 +1,31 @@
+}
+
+
+.actions {
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  color: #ccc;
+}
+
+.action {
+  margin-left: 2p0;
```

Biome (1.9.4)

[error] Unexpected unknown unit: p0

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)